### PR TITLE
Translate En to Ja and Update a few hash values

### LINKED ISSUE #127: Titles at all levels and notes of some Japanese documents have not yet been translated

### PR DESCRIPTION
**Pull request by GreasySlug**
fix #127.

Some titles and comments were not translated into Japanese, so those have been translated into Japanese.

Updated where the hash value is different.

Note: The parentheses in the title may refer to annotations or simply to Japanese translations, but they have been truncated for consistency.
Also, I think the API is still in place, so I have not made any fixes.

@mtshiba

### CODE REVIEW

File: doc/JA/syntax/22_subroutine.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# Subroutine signatures
+# サブルーチンシグネチャ
 
-[![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3Ddoc/EN/syntax/22_subroutine.md%26commit_hash%3D51de3c9d5a9074241f55c043b9951b384836b258)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=doc/EN/syntax/22_subroutine.md&commit_hash=51de3c9d5a9074241f55c043b9951b384836b258)
+[![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3Ddoc/EN/syntax/22_subroutine.md%26commit_hash%3Dddcbe9859e56d63bb3dd001be216cd4343e1771e)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=doc/EN/syntax/22_subroutine.md&commit_hash=ddcbe9859e56d63bb3dd001be216cd4343e1771e)
 
-## Func
+## 関数
```
Comment (GreasySlug) [inline doc/JA/syntax/22_subroutine.md:5]: Do you use Func (Proc) as an abbreviation for Function, or do you have a different intention?

File: doc/JA/syntax/type/13_algebraic.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+# 代数演算型
 
 [![badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com%2Fdefault%2Fsource_up_to_date%3Fowner%3Derg-lang%26repos%3Derg%26ref%3Dmain%26path%3Ddoc/EN/syntax/type/13_algebraic.md%26commit_hash%3Dc120700585fdb1d655255c8e2817bb13cc8d369e)](https://gezf7g7pd5.execute-api.ap-northeast-1.amazonaws.com/default/source_up_to_date?owner=erg-lang&repos=erg&ref=main&path=doc/EN/syntax/type/13_algebraic.md&commit_hash=c120700585fdb1d655255c8e2817bb13cc8d369e)
 
 代数演算型は、型を代数のようにみなして演算することで生成される型のことです。
 代数演算型が扱う演算は、Union, Intersection, Diff, Complementなどがあります。
 通常のクラスはUnionのみが行えて、他の演算は型エラーになります。
 
-## Union
+## 合併
```
Comment (GreasySlug) [inline doc/JA/syntax/type/13_algebraic.md:9]: I used translations of mathematical set theory.

File: doc/JA/syntax/22_subroutine.md (hunk 1 above)
Comment (mtshiba) [reply doc/JA/syntax/22_subroutine.md:5]: For this part, it is an abbreviation.
The `Func` and `Proc` in the code refer to the type of function and procedure, respectively (just as the type of interger is `Int`).

**Comment by mtshiba** [comment]
Thank you!

**Pull request merged**